Pcap++/src/KniDeviceLinux.cpp

```diff
@@ -0,0 +1,279 @@
+#if defined(USE_DPDK) && defined(LINUX)
```

Comment:
I have already stated my position here:
https://github.com/seladb/PcapPlusPlus/issues/192#issuecomment-489420071

Reply:
I'm not sure I understand this comment:
> Implementation is split in two files bacause DPDK and Linux kernel have incompatible definitions.

what is incompatible between Linux and DPDK?

Reply:
Most of them lay in <net/*> and <netinet/*> Linux include paths because DPDK unceremoniously include parts of their net protocol implementations (and their types) inside other unrelated headers and some in <sys/types.h>. 
As my practice with DPDK gose to separate the DPDK and Linux parts of code is a good decision because there may be a cese where you forgot to include corresponding Linux or DPDK genaral types and one from other will take their place.

Reply:
@seladb I want to hear Your thoughts about it. Do we leave this as it is or it is must be merged with Kni device implementation?

Reply:
For the sake of simplicity I think it'd be better if we put everything in one file. Where possible I always prefer code which is straight-forward and simple

Reply:
\+